modular_skyrat/modules/primitive_fun/code/glassblowing.dm

```diff
@@ -201,6 +201,9 @@
 				if("Lens")
 					find_glass.chosen_item = /obj/item/glassblowing/glass_lens
 					find_glass.required_actions = list(0,0,3,3,3) //paddling, shearing, jacking
+				if("Bottle")
+					find_glass.chosen_item = /obj/item/reagent_containers/cup/glass/bottle/small
+					find_glass.required_actions = list(3,2,3,0,0)
```

Comment:
add the comment describing the steps that the others do (holy shit this really should've been done differently in the first place)

Reply:
I swear ill refactor all this later